packages/components/src/tree-grid/stories/index.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TreeGrid, { TreeGridRow, TreeGridCell } from '..';
+import { Button } from '../../button';
+
+const meta: ComponentMeta< typeof TreeGrid > = {
+	title: 'Components (Experimental)/TreeGrid',
+	component: TreeGrid,
+	subcomponents: { TreeGridRow, TreeGridCell },
+	argTypes: {
+		children: { control: { type: null } },
+	},
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
```

Comment:
Note that this won't fire actions for `onExpandRow` until we add a specific story that allows it to happen. I'm not going to do that in this PR because it is somewhat non-trivial.

Reply:
What is the specific config that we need to replicate? Is it worth adding it to the `onExpandRow` prop docs?

Reply:
I think a good example would need to implement a simple version of a [file browser](http://wwwendt.de/tech/fancytree/demo/sample-aria-treegrid.html) where you can expand folder rows. Most of the functionality isn't baked into the component so we'd need to manage it ourselves (toggling visibility, state, styling, etc). We can write a better description for `onExpandRow` once we better understand what it entails.